applications/IgaApplication/custom_processes/map_nurbs_volume_results_to_embedded_geometry_process.h

```diff
@@ -43,13 +43,11 @@ class KRATOS_API(IGA_APPLICATION) MapNurbsVolumeResultsToEmbeddedGeometryProcess
     /// Pointer definition of MapNurbsVolumeResultsToEmbeddedGeometryProcess
     KRATOS_CLASS_POINTER_DEFINITION(MapNurbsVolumeResultsToEmbeddedGeometryProcess);
 
-    typedef Node                                             NodeType;
+    typedef Node                                                NodeType;
```

Comment:
That looks a bit weird on the phone. Would be alright though.

Reply:
I just put NodeType to the correct indent.